app/middlewares/admin_permissions.js

```diff
@@ -0,0 +1,5 @@
+const errors = require('../errors');
+
+exports.handle = (req, res, next) => {
+  req.currentUser.role === 'ADMIN' ? next() : next(errors.notAdminError);
```

Comment:
it would be nice to have this stored as constants somewhere in case the list of possible roles grows bigger, a helpers with an object is fine, or if you prefer to have a specific file for definitions or something like that it would be fine too.